CMakeLists.txt

```diff
@@ -201,6 +211,14 @@ add_dependencies(psi4_core ambit_external
                            libint_external
                            pcmsolver_external
                            pybind11_external)
+if(ENABLE_psi4pasture)
+  ExternalProject_Add(psi4pasture
+    GIT_REPOSITORY https://github.com/amjames2/psi4pasture
```

Comment:
Moving this to external/pasture/CMakeLists.txt works fine, as long as I leave it off of https://github.com/psi4/psi4/blob/master/external/CMakeLists.txt and instead put `add_subdirectory(external/pasture)` here. 

I put the pasture block here instead since it seems, everything in external is something that could be built without psi4 existing. Pasture is a slightly different beast and the dependency goes the other way.